build.gradle

```diff
@@ -32,19 +37,23 @@ subprojects {
   }
 
   // Common dependencies
-  ext.springVersion = '4.3.12.RELEASE'
-  ext.springBootVersion = '1.5.9.RELEASE'
+  ext.compileWithSpringVersion = '5.0.6.RELEASE'
+  ext.compileWithBootVersion = '2.0.2.RELEASE'
+
+  ext.testWithSpringVersion = '4.3.13.RELEASE'
+  ext.testWithBootVersion = '1.5.9.RELEASE'
```

Comment:
The latest release of the 1.x series is now 1.5.13, should we upgrade this or is that considered out of scope for this PR?

Reply:
This would seem sane.. I'll check & push an update if all is good.

Reply:
Updated to 4.3.17/1.5.13